rust/src/lib/ovs.rs

```diff
@@ -33,7 +32,7 @@ impl OvsDbGlobalConfig {
 
     // User want to remove all settings except OVN.
     pub(crate) fn is_purge(&self) -> bool {
-        self.prop_list.is_empty()
+        self.external_ids.is_none() && self.other_config.is_none()
```

Comment:
How do we express this at yaml ? I think previously it was something like `external-ids: {}` so this breaks compatiblity.

Reply:
Fixed and also included unit test cases for this.

To purge ovs-db config, you may:

```yaml
---
ovs-db: {}
```

or

```yaml
---
ovs-db:
  external_ids: {}
  other_config: {}
```